### ISSUE TITLE
PyVCF assigns a QUAL value of '.' instead of zero

### ISSUE THREAD

**Issue by knewl** [artic-network/fieldbioinformatics#134]
Hello,

This is an [issue](https://github.com/jamescasbon/PyVCF/issues/329) in PyVCF where instead of a zero a '.' is written to the QUAL field of a VCF. The issue was raised a few years ago and is therefore unlikely to be addressed soon. 

For example, this variant with QUAL of 0.0 in sample_name.1.vcf

`MN908947.3      7576    .       T       TA      0.0     PASS    DP=2.0;DPS=0.0,2.0      GT:GQ   1:0`

has a QUAL value of '.' once it is merged into sample_name.merged.vcf

`MN908947.3      7576    .       T       TA      .       PASS    DP=2.0;DPS=0.0,2.0;Pool=1       GT:GQ   1:0`

This causes an error in vcf_filter.py when it attempts to assess whether the quality is < 20:

`TypeError: '<' not supported between instances of 'NoneType' and 'int'`

My suggestion would be to ignore variants in vcf_filter.py where v.QUAL is not numeric.
